package-lock.json

```diff
@@ -1952,6 +1952,20 @@
 			"integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
 			"dev": true
 		},
+		"node_modules/fsevents": {
```

Comment:
**Question** Is this in here on accident? I think we should try keep this out of the PR if we don't need this.

Reply:
It happened automatically after making changes I'm scss and rebuilding the project

Reply:
It happened in #166 as well

Reply:
Now when I look at it more closely then what removed in #166 got restored in this one. I suggest you take this branch delete package-lock.json file and rebuild to see if these changes should be there or not.

Reply:
I am not sure I understand what you want me to do here? Can you not remove these changes from the PR? Or should that have never been removed?

Reply:
Let's drop them for the time being. Maybe they shouldn't be there.